src/routers/orders.js

```diff
@@ -0,0 +1,24 @@
+const ordersRouter = require('express').Router();
+const { OrdersController } = require('../controllers');
+const { createOrderPayloadValidator } = require('../middlewares/validators');
+const { updateOrderPayloadValidator } = require('../middlewares/validators');
+const { idParamValidator } = require('../middlewares/validators');
+const { getOrdersQueryValidator } = require('../middlewares/validators');
+
+const ordersController = new OrdersController();
+
+ordersRouter.post(
+  '/',
+  createOrderPayloadValidator,
+  ordersController.createOrder
+);
+ordersRouter.get('/', getOrdersQueryValidator, ordersController.getOrders);
```

Comment:
Get orders endpoint: only admin or super admin can use it. It must contain skip and take params. It must return {list: [], total_size: number} object

Reply:
Fixed